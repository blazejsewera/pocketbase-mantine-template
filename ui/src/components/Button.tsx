import { Button as MButton } from '@mantine/core'
export const Button = () => <MButton>Hello</MButton>
