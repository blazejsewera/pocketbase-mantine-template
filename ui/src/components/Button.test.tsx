import { Button } from './Button'
import { snap } from './test/snapshot'

it('renders properly', () => {
  const s = snap(<Button />)
  expect(s).toMatchSnapshot()
})
