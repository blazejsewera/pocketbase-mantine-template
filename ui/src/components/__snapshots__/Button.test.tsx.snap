// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<button
  className="mantine-UnstyledButton-root mantine-Button-root mantine-8nr514"
  data-button={true}
  type="button"
>
  <div
    className="mantine-3xbgk5 mantine-Button-inner"
  >
    <span
      className="mantine-qo1k2 mantine-Button-label"
      style={
        {
          "textTransform": undefined,
        }
      }
    />
  </div>
</button>
`;
